src/ZoneData.cs

```diff
@@ -201,10 +202,10 @@ private static void ExtractNpcs(List<string> npcList, int mapId, MapDataFile ifo
 
     private static void ExtractWarpGates(List<string> warpList, int mapId, MapDataFile ifo)
     {
-      const string warpStb = "./3DDATA/stb/warp.stb";
-      const string zoneStb = "./3DDATA/stb/list_zone.stb";
-      const string warpGateModel = "./3DDATA/special/warp_gate01/warp.zms";
-      const string decoSpecialList = "./3DDATA/special/list_deco_special.zsc";
+      const string warpStb = Globals.stbroot + "warp.stb";
+      const string zoneStb = Globals.stbroot + "list_zone.stb";
+      const string warpGateModel = Globals.stbroot + "../special/warp_gate01/warp.zms";
+      const string decoSpecialList = Globals.stbroot +"../special/list_deco_special.zsc";
```

Comment:
Personally I don't like this as this would resolve to "../3DDATA/stb/../special/warp_gate01/warp.zms" and "../3DDATA/stb/../special/list_deco_special.zsc".

Also this tool is being deprecated in favor for https://github.com/dev-osrose/rose_data_converter